Automattic/wp-calypso
# Load middleware async

#### Changes proposed in this Pull Request
We are adding [redux-dyamic-middlewares](https://github.com/pofigizm/redux-dynamic-middlewares) package, which would allow us to have a very light "placeholder" like middleware which can be used to add/remove middlewares based on our needs, in this case we would want to only load the lasagna middleware if the reader section is loaded. 

More context https://github.com/Automattic/wp-calypso/pull/40396

#### Testing instructions
- ICFY should only add about 500B (which would result in removing about 11KB that were added with the lasagna middleware) to the main entry point, that would be the 3rd party dependency that allows us to lazy load middlewares
- Enable `localStorage.debug="lasagna:*"`
- Go to reader -> in console we should see lasagna logs about connecting to the socket
- Realtime comments should still work in full post view

Fixes https://github.com/Automattic/wp-for-teams/issues/90
